contrib/symlink-githooks.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+remove_symlinks() {
+	for hook in githooks/*
+	do
+		bn=$(basename "$hook")
+		echo "Removing git hook symlink $bn"
+		rm ".git/hooks/$bn"
+	done
+	exit 0
+}
+
+while getopts "r" flag; do
+	case $flag in
+		r) remove_symlinks
+		;;
+		*) exit 1
+		;;
+	esac
+done
+
+for hook in githooks/*
+do
+	bn=$(basename "$hook")
+	if [ ! -f ".git/hooks/$bn" ]; then
+		echo "Symlinking $bn..."
+		ln -s "../../githooks/$bn" ".git/hooks/$bn"
+	else
+		echo "$bn hook already found, skipping"
+	fi
+done
```

Comment:
It'd be nicer to have both as functions.

Also is the hooks directory guaranteed to exist? It might be good idea to run `mkdir -p`/`rmdir --ignore-fail-on-non-empty`.

Reply:
I did a quick check and on my system `.git/hooks` exists after running `git init`. This whole script is only for newer devs anyways so I wouldn't be too concerned with setups outside the ordinary (for example script does not work with git trees).

Reply:
I created an `add_symlinks` function to handle the primary case.